packages/jaeger-ui/src/components/SearchTracePage/SearchResults/ScatterPlot.js

```diff
@@ -36,7 +36,11 @@ function ScatterPlotImpl(props) {
         width={containerWidth}
         height={200}
       >
-        <XAxis title="Time" tickTotal={4} tickFormat={t => moment(t).format('hh:mm:ss a')} />
+        <XAxis
+          title="Time"
+          tickTotal={4}
+          tickFormat={t => moment(t / ONE_MILLISECOND).format('hh:mm:ss a')}
```

Comment:
`moment` - is the main function of moment.js framework - https://momentjs.com/

It may accept different inputs (strings, Date objects, timestamps). Here it accepts timestamp (number of milliseconds since 1 January 1970 UTC)

Reply:
Thanks.

While this may be an OK fix for this specific issue, I think a bigger problem is that we have to do that in the first place. I would expect the UI to have some domain model of the trace and expose the timestamps not in raw microseconds format, but in some structured representation, so that the presentation code wouldn't have to do these conversions all over the place.